perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -446,6 +446,11 @@ sub _missed_collection_reports {
     my $config = $self->feature('whitespace');
     my $state_mapping = $config->{missed_collection_state_mapping};
 
+    # Create a lookup hash mapping fms_state to text
+    my %fms_state_to_text = map {
```

Comment:
As you don't need the keys, you could have `map { $_->{fms_state} => $_->{text} } values %$state_mapping` here.